tests/unit/queryBuilder/QueryBuilder.js

```diff
@@ -666,7 +666,7 @@ describe('QueryBuilder', () => {
       })
       .runBefore(() => {
         expect(mockKnexQueryResults[0]).to.equal(1);
-        return Bluebird.delay(1).return(++mockKnexQueryResults[0]);
+        return Bluebird.delay(1).then(() => ++mockKnexQueryResults[0]);
```

Comment:
here logic changed, because mockQueryResults[0] was incremented before delay, now after

Reply:
Isn't it more correct to increment it after delay, if we are simulating query happening during that delay?

Reply:
Yes, you are right